openedx/core/djangoapps/content_tagging/tests/test_objecttag_export_helpers.py

```diff
@@ -441,7 +441,7 @@ def test_build_library_object_tree(self) -> None:
         """
         Test if we can export a library
         """
-        with self.assertNumQueries(8):
+        with self.assertNumQueries(11):
```

Comment:
I think this query count will drop back down after making the [changes recommended above](https://github.com/openedx/edx-platform/pull/35476/files#diff-a760c3e87f2f671ac2453abbbaf6f10ca052c7acf1c46c1d36f24646e1d52a00R243), but if it doesn't: can we prefetch the draft data with `select_related('publishable_entity__draft__version')`, cf [`component.versioning.draft` comment](https://github.com/openedx/openedx-learning/blob/6fa83ba94c32fa6a1d2abddfe666ffcb0c628a36/openedx_learning/apps/authoring/publishing/model_mixins.py#L152-L179)?

Reply:
You're right! Thank you! d4810dccc7510063d1ea2138dc72ed4e637f56b7